lndclient/lnd_services.go

```diff
@@ -12,20 +11,33 @@ import (
 	"github.com/btcsuite/btcutil"
 	"github.com/lightninglabs/loop/swap"
 	"github.com/lightningnetwork/lnd/lncfg"
+	"github.com/lightningnetwork/lnd/lnrpc/verrpc"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials"
 )
 
-var rpcTimeout = 30 * time.Second
+var (
+	rpcTimeout = 30 * time.Second
+
+	// minimalCompatibleVersion is the version of lnd we require at least
+	// when checking compatibility. This will be used as a fall back version
+	// if none is specified in the configuration.
```

Comment:
Maybe add that this is the minimum required version if you want to be able to use all functionality. When only a subset is required, a lower compatible version can be specified.